application/views/survey_responses_view.py

```diff
@@ -44,10 +45,12 @@ def add_survey_response(case_id):
 
         upload_success = survey_response.add_survey_response(case_id, file, file_name, survey_ref)
 
+        if upload_success == "False":
+            return make_response(FILE_TOO_SMALL, 400)
+
```

Comment:
This feels a bit wrong as you're first checking that the upload_sucess variable does not match the string 'false' and then immediately afterwards it's checked as a boolean.